client/elements/text/sc-segmented-text.html

```diff
@@ -192,6 +192,10 @@ <h3>Could not find the sutta text.</h3>
                   segmentedIDsAdded: {
                       type: Boolean,
                       value: false
+                  },
+                  scriptChanged: {
```

Comment:
The name sounds more like an event. Maybe change it to something like 'hasScriptChanged'?